src/utils/conversions.ts

```diff
@@ -1,50 +0,0 @@
-import { NonEmptyString } from "italia-ts-commons/lib/strings";
```

Comment:
why delete this?

Reply:
I found this methods not to be used. Moreover, the linter now asks me to strip manual casting off, so what these methods are is actually just `as ModelId`